chapter3.adoc

```diff
@@ -57,7 +57,7 @@ Trusted entities like RoT should configure IOPMP or equivalent protection before
 
 === Security Fault Error Reporting
 
-A dedicated error code, security fault error (cmderr 6), is included in `cmderr` of abstractcs (at 0x16 in Debug Module). Misconfigurations of the dcsr and issuance of abstract commands under disallowed circumstance can signify such an error. Additionally, the bus security fault error (sberror 6) is introduced in `sberror` of sbcs (at 0x38 in Debug Module) to denote errors related to system bus access. 
+A dedicated error code, security fault error (cmderr 6), is included in `cmderr` of abstractcs (at 0x16 in Debug Module). Issuance of abstract commands under disallowed circumstance sets `cmderr` to 6. Additionally, the bus security fault error (sberror 6) is introduced in `sberror` of sbcs (at 0x38 in Debug Module) to denote errors related to system bus access. 
```

Comment:
Missing some backticks on register names (abstractcs, sbcs).  CSR field names don't need to be in backticks, they are generally all caps but the debug spec predates this convention.  So fine to keep the backticks on those if you prefer that.